backend/app/tests/services/inquiry/test_create_inquiry_service.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from pydantic import ValidationError
+from sqlalchemy.exc import IntegrityError
+from sqlmodel import Session
+
+from app.models.inquiry import MAX_LENGTH, MIN_LENGTH, InquiryCreate
+from app.services.inquiries import create_inquiry
+
+
+def test_inquiry_service_create_should_create_inquiry_when_inquiry_does_not_exist(
+    db: Session,
+) -> None:
+    text = "Test Inquiry"
+    inquiry_data = InquiryCreate(text=text)
+    result = create_inquiry(session=db, inquiry_in=inquiry_data)
+    assert result.text == text
+    assert result.id is not None
+    assert result.created_at is not None
+
+
+def test_create_inquiry_service_should_raise_error_when_no_text_parameter_is_given(
+    db: Session,
+) -> None:
+    with pytest.raises(ValidationError, match=r"Field required"):
+        inquiry_data = InquiryCreate()
+        create_inquiry(session=db, inquiry_in=inquiry_data)
+
+    with pytest.raises(ValidationError, match=r"Field required"):
+        inquiry_data = InquiryCreate(taxed="Taxed inquiry")
```

Comment:
What's taxed? I don't recall our Inquiry SQL model having this field.

Reply:
yes. it is testing for failure.  the real field we have is `text`